src/policy/fees.cpp

```diff
@@ -957,12 +956,16 @@ void CBlockPolicyEstimator::FlushFeeEstimates()
     }
 }
 
+// The current format written, and the version required to read. Must be
+// increased to at least 279900+1 on the next breaking change.
+constexpr int CURRENT_FILE_VERSION{149900};
```

Comment:
`static` ? Move it to the top of the file?

Could consider defining three values:

```
static constexpr int FEE_FILE_MIN_READ_VERSION = 149900;
static constexpr int FEE_FILE_MAX_READ_VERSION = 279900;
static constexpr int FEE_FILE_WRITE_VERSION = 149900; 
```